python/ql/lib/semmle/python/dataflow/new/internal/DataFlowPrivate.qll

```diff
@@ -2,6 +2,7 @@ private import python
 private import DataFlowPublic
```

Comment:
I think you are missing `allowParameterReturnInSelf`, see https://github.com/github/codeql/blob/main/ruby/ql/lib/codeql/ruby/dataflow/internal/DataFlowPrivate.qll#L940.

Reply:
Thanks! Indeed we just have `none()`.